custom_components/tuya_local/devices/inkbird_ibt_26s_thermometer.yaml

```diff
@@ -0,0 +1,292 @@
+name: BBQ thermometer
+products:
+  - id: h8qbni4ouwn5gjh0
+    name: Inkbird IBT-26S
+primary_entity:
+  entity: sensor
+  name: Temperature Probe 1
+  class: temperature
+  dps:
+    - id: 107
+      type: integer
+      name: sensor
+      class: measurement
+      mapping:
+        - scale: 100
+      unit: F
+secondary_entities:
+  - entity: sensor
+    name: Temperature Probe 2
+    class: temperature
+    dps:
+      - id: 108
+        type: integer
+        name: sensor
+        class: measurement
+        optional: true
+        mapping:
+          - scale: 100
+        unit: F
+  - entity: sensor
+    name: Temperature Probe 3
+    class: temperature
+    dps:
+      - id: 109
+        type: integer
+        name: sensor
+        class: measurement
+        optional: true
+        mapping:
+          - scale: 100
+        unit: F
+  - entity: sensor
+    name: Temperature Probe 4
+    class: temperature
+    dps:
+      - id: 110
+        type: integer
+        name: sensor
+        class: measurement
+        optional: true
+        mapping:
+          - scale: 100
+        unit: F
+  - entity: sensor
+    name: Temperature Probe 5
+    class: temperature
+    dps:
+      - id: 124
+        type: integer
+        name: sensor
+        class: measurement
+        optional: true
+        mapping:
+          - scale: 100
+        unit: F
+  - entity: sensor
+    name: Temperature Probe Oven
+    class: temperature
+    dps:
+      - id: 125
+        type: integer
+        name: sensor
+        class: measurement
+        optional: true
+        mapping:
+          - scale: 100
+        unit: F
+  - entity: sensor
+    name: Internal temperature
+    class: temperature
+    category: diagnostic
+    dps:
+      - id: 112
+        type: integer
+        name: sensor
+        unit: C
+        class: measurement
+
+  - entity: select
+    translation_key: temperature_unit
+    category: config
+    dps:
+      - id: 19
+        type: string
+        name: option
+        mapping:
+          - dps_val: F
+            value: fahrenheit
+          - dps_val: C
+            value: celsius
+  - entity: sensor
+    class: battery
+    category: diagnostic
+    dps:
+      - id: 101
+        type: integer
+        name: sensor
+        unit: "%"
+  - entity: number
```

Comment:
May be better emulated as a light.

Reply:
I have adjusted the range minimum to 5% instead of 0%, is there a light entity without an on and off switch integrated?

Reply:
Yes, lights can have just a brightness dp. If the brightness does not go down to 0, they will not be presented with a switch in the UI.

Reply:
There is also `translation_key: backlight` or `translation_key: display` which can be used with light in place of an explicit name and icon.

Reply:
I've tried that several times, it's always presented as a slider with a switch in the UI. I changed it anyway, maybe I did something wrong.